### PR TITLE
Hotkeys

### PR DESCRIPTION
**Pull request by isabellakutger**
This PR adds hotkeys to the new admin UI, which can be viewed in the hotkey cheatsheet.
To make the hotkeys work a library is used and some minor changes in comparison to the old UI were made.

Changes and notes regarding the hotkeys:
- no hotkey is supported globally (e.g. you can not use 'n' everywhere to add a new series)
> - non of the tested libraries supported this
- There is no hotkey for previous and next table filters
> - currently we don't save the filter history
- the remove filter hotkey works in every table
- the hotkey of the cheatsheet was changed from `?` to `h`
> - the library needs shift as an extra key in the key sequence of a hotkey
> - '?' has different key combinations depending on key layout
- Player hotkeys are shown in the cheatsheet, however, they have not been implemented yet since there is currently no player

### CODE REVIEW

File: modules/admin-ui-picard/app/src/components/shared/TableFilters.js [hunk 1]
```diff
@@ -118,80 +120,85 @@ const TableFilters = ({loadingFilters, filterMap, textFilter, selectedFilter, se
...
+                            }
+
+                            {/* Show for each selected filter a blue label containing its name and option */}
+                            {
+                                filterMap.map((filter, key) => {
+                                    if(!!filter.value) { return (
+                                        <span className="ng-multi-value" key={key}>
                                         <span>
                                             {
                                                 // Use different representation of name and value depending on type of filter
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/components/shared/TableFilters.js]: Please fix inconsistent indentation

File: modules/admin-ui-picard/app/src/components/shared/TableFilters.js [hunk 2]
```diff
@@ -206,50 +213,52 @@ const TableFilters = ({loadingFilters, filterMap, textFilter, selectedFilter, se
...
+                                            <a title={t('TABLE_FILTERS.REMOVE')} onClick={() => removeFilter(filter)}>
                                             <i className="fa fa-times"/>
                                           </a>
                                     </span>
-                                )
-                            }})
-                        }
+                                    )
+                                    }})
+                            }
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/components/shared/TableFilters.js]: Please fix inconsistent indentation.

File: modules/admin-ui-picard/app/src/configs/hotkeysConfig.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            allowIn: ['INPUT', 'SELECT', 'TEXTAREA']
+        },
+        PREVIOUS_DASHBOARD_FILTER: {
+            name: 'select_previous_dashboard_filter',
+            description: 'HOTKEYS.DESCRIPTIONS.GENERAL.SELECT_PREVIOUS_DASHBOARD_FILTER',
+            combo: ['F'],
+            sequence: 'F',
+            action: 'keyup',
+            allowIn: ['INPUT', 'SELECT', 'TEXTAREA']
+        },*/
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/configs/hotkeysConfig.js:67]: will these be added in the future?

File: modules/admin-ui-picard/app/src/components/shared/HotKeyCheatSheet.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+import { getApplicationKeyMap } from 'react-hotkeys';
+
+/**
+ * This component renders the hotkey cheat sheet showing all available hotkeys
+ */
+const HotKeyCheatSheet = ({ close }) => {
+    const { t } = useTranslation();
+
+    console.log("keys");
+    console.log(getApplicationKeyMap());
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/components/shared/HotKeyCheatSheet.js]: remove `console.log()` or use logger instead

File: modules/admin-ui-picard/app/src/components/shared/HotKeyCheatSheet.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                                                                <>
+                                                                    <span key={key}>
+                                                                        <span className="key">{t('HOTKEYS.KEYS.' + comboKey.toUpperCase(), comboKey)}</span>
+                                                                    </span>
+                                                                    {comboKey === availableHotkeys[hotkeyGroup][hotkey].combo[availableHotkeys[hotkeyGroup][hotkey].combo.length - 1]
+                                                                        ? ''
+                                                                        : ' + '}
+                                                                </>
+                                                            ))}
+                                                    </span>
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/components/shared/HotKeyCheatSheet.js]: indent further (one tabulator)

**Review by SimoneSchmidt — CHANGES_REQUESTED** [review]

File: modules/admin-ui-picard/app/src/configs/hotkeysConfig.js (hunk 3 above)
Comment (isabellakutger) [reply modules/admin-ui-picard/app/src/configs/hotkeysConfig.js:67]: Yes, but we have to change some other things before these can be added

**Review by SimoneSchmidt — APPROVED** [review]

**Pull request merged**